src/components/Navigation/Nav.tsx

```diff
@@ -97,21 +100,27 @@ export const Nav = ({ className }: NavProps) => {
       <nav className={classes.nav}>
         <ul className={classes.list}>
           <li className={classes.listItem}>
-            <Link to={ROUTE.home}>Inicio</Link>
+            <NavLink className={isActiveLink} to={ROUTE.home}>
+              Inicio
+            </NavLink>
           </li>
           <li>
             <span className={classes.dots}></span>
           </li>
 
           <li className={classes.listItem}>
-            <Link to={ROUTE.projects}>Proyectos</Link>
+            <NavLink to={ROUTE.projects} className={isActiveLink}>
+              Proyectos
+            </NavLink>
```

Comment:
```suggestion
             <li>
            <NavLink to={ROUTE.projects} className={({ isActive }) => classes.listItem(isActive)}>
              Proyectos
            </NavLink>
          </li>
```

Lo mismo de mas arriba